### ISSUE TITLE
md-list-item + links versus buttons

### ISSUE THREAD

**Issue by 0101adm** [angular/material#6507]
the padding on icons differ from links and buttons.

![screen shot 2015-12-30 at 6 32 24 pm](https://cloud.githubusercontent.com/assets/5133412/12060040/e3a53f32-af23-11e5-8b65-a45369b4856c.png)

```
md-list
				md-list-item(ng-click='')
					md-icon(md-svg-icon='communication:phone')

					p
						| Call Me

				md-list-item(ng-click='')
					md-icon(md-svg-icon='communication:email')

					p
						| Email Me



				md-list-item(ui-sref='map')
					md-icon(md-svg-icon='maps:map-marker-radius')

					p
						| Near Me

				md-list-item(ui-sref='map')
					md-icon(md-svg-icon='maps:map')

					p
						| Map Search
```

**Comment by DevVersion** [comment]
This seems to be a dupe of #6317, and there is already a fix #6496

**Comment by 0101adm** [comment]
:+1:
